lib/bulk/common.js

```diff
@@ -208,21 +216,23 @@ var BulkWriteResult = function(bulkResult) {
  * @property {string} errmsg Write concern error message.
  * @return {WriteConcernError} a WriteConcernError instance
  */
-var WriteConcernError = function(err) {
-  if (!(this instanceof WriteConcernError)) return new WriteConcernError(err);
+class WriteConcernError {
+  constructor(err) {
+    if (!(this instanceof WriteConcernError)) return new WriteConcernError(err);
```

Comment:
you no longer need to do this check when using class syntax, JavaScript will not allow you to create a class instance without `new`.